scipy/stats/tests/test_distributions.py

```diff
@@ -5238,6 +5238,39 @@ def test_crystalball_entropy():
     assert_allclose(res1, res2, rtol=1e-7)
 
 
+def test_invweibull():
```

Comment:
This passes in master for me. Would be preferable to show that overriding `_fitstart` improves the fit.

Reply:
Never mind, that was the point.

Reply:
I don't think there is necessarily a reason to believe that it does improve the fit.  Fixing the warning is the main goal of the PR.  I was working with `invweibull` over in https://github.com/scipy/scipy/pull/13699, and the spurious warning caused the test suite to fail.

Reply:
Yup, I read more closely a minute later.